_sass/foundation-components/_accordion.scss

```diff
@@ -98,24 +111,31 @@ $accordion-content-active-bg-color: $white !default;
 //    @params $bg [ color  ]: Background color for the content when it's visible
 //    @params $active_class [ string ]: Class name used to keep track of which accordion tab should be visible.
 
-@mixin accordion-content($bg: $accordion-content-active-bg-color, $padding: $accordion-content-padding, $active_class: 'active'){
+@mixin accordion-content($bg: $accordion-content-active-bg-color, $padding: $accordion-content-padding, $active_class: 'active') {
   display: none;
-  @if type-of($padding) != "number" {
+
+  @if type-of($padding) !="number" {
     @warn "#{$padding} was read as a #{type-of($padding)}";
-    @if $accordion-content-padding != null {
+
+    @if $accordion-content-padding !=null {
       @warn "`#{$padding}` isn't a valid number. $accordion-content-padding used instead";
       padding: $accordion-content-padding;
-    } @else {
+    }
+
+    @else {
```

Comment:
this looks like "technically correct" formatting, but the original one is more traditional for anyone reading these files.

Reply:
Same as for the other one just the default sass plugin in vscode I have, doesn't seem very good to me.